iOS/Flows/Home/_New/ConversationViewControllerDataSource.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  ConversationViewControllerDataSource.swift
+//  ConversationViewControllerDataSource
+//
+//  Created by Martin Young on 9/15/21.
+//  Copyright © 2021 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+import StreamChat
+
+typealias ConversationCollectionSection = ConversationCollectionViewDataSource.SectionType
+typealias ConversationCollectionItem = ConversationCollectionViewDataSource.ItemType
+
+class ConversationCollectionViewDataSource: CollectionViewDataSource<ConversationCollectionViewDataSource.SectionType,
```

Comment:
The conversation data source. A subclass of our base DataSource type.

This particular interaction doesn't support threading, but I finally figured out how to do it right and it'll be in another PR.